src/main/scala/org/scalajs/tools/tsimporter/parser/TSDefParser.scala

```diff
@@ -104,7 +104,7 @@ class TSDefParser extends StdTokenParsers with ImplicitConversions {
     "enum" ~> typeName ~ ("{" ~> ambientEnumBody <~ "}") ^^ EnumDecl
 
   lazy val ambientEnumBody: Parser[List[Ident]] =
-    repsep(identifier <~ opt("=" ~ (numericLit | stringLit) ), ",") <~ opt(",")
+    repsep(identifier <~ opt("=" ~ ((opt("-") ~> numericLit) | stringLit) ), ",") <~ opt(",")
```

Comment:
Why is this change necessary, given that there is also a change at the source, in `numericLiteral`?

Reply:
Oops. This was accidentally commited and is not required.
I remove this.